lookout/style/format/benchmarks/evaluate_smoke.py

```diff
@@ -218,28 +218,25 @@ def calc_metrics(bad_style_code: str, correct_style_code: str, fe: Optional[Feat
     :param url: Repository url if applicable. Useful for more informative warning messages.
     :param commit: Commit hash if applicable. Useful for more informative warning messages.
 
-    :return: A dictionary with losses and predicted code for "global" and "local" indentation \
-             strategies.
+    :return: A dictionary with losses and predicted code.
     """
     # TODO (zurk): make optional arguments non-optional.
     if (vnodes is None) ^ (fe is None):
         raise ValueError("vnodes and fe should be both None or not None.")
-    losses = {}
-    for indentation in ("global", "local"):
-        if fe is not None:
-            predicted_code = CodeGenerator(fe, skip_errors=True, url=url, commit=commit).generate(
-                vnodes, indentation)
-        else:
-            predicted_code = bad_style_code
-        misdetection, undetected, detected_wrong_fix, detected_correct_fix = \
-            calc_aligned_metrics(*align3(bad_style_code, correct_style_code, predicted_code))
-        losses[indentation] = {
-            "misdetection": misdetection,
-            "undetected": undetected,
-            "detected_wrong_fix": detected_wrong_fix,
-            "detected_correct_fix": detected_correct_fix,
-            "predicted_file": predicted_code,
-        }
+    if fe is not None:
```

Comment:
I see

```python
# TODO (zurk): make optional arguments non-optional.
```

How about fixing this TODO in this PR?

Reply:
it is unrelated. I prefer to do it in another one after I finish with CodeGenerator